scipy/special/tests/test_gammainc.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import division
```

Comment:
Best to use the full standard stanza
`from __future__ import division, absolute_import, print_function`
so that all files share the same semantics.